docs/sim/cli.rst

```diff
@@ -0,0 +1,132 @@
+SMARTS Command Line 
+====================
+
+SMARTS provides a command-line tool to interact with scenario studio and Envision.
```

Comment:
I am not entirely certain we need this file since the commands can be checked in `scl`. If we keep around this file, it should be auto-generated from `scl`.

Reply:
What I mean here is that it can go out of date very easily and then provide the wrong information.